### PR TITLE
Migrate listings to action group

### PR DESCRIPTION
**Pull request by Hofer-Julian**

### CODE REVIEW

File: book/listings/actions/1/main.rs [hunk 1]
```diff
@@ -31,11 +30,12 @@ fn build_ui(app: &Application) {
...
-    action_close.connect_activate(clone!(@weak window => move |_, _| {
-        window.close();
-    }));
-    window.add_action(&action_close);
+    let action_close = ActionEntry::builder("close")
+        .activate(|window: &ApplicationWindow, _, _| {
+            window.close();
+        })
+        .build();
+    window.add_action_entries([action_close]);
```
Comment (bilelmoussaoui) [inline book/listings/actions/1/main.rs:38]: The action exists already?

Comment (Hofer-Julian) [reply book/listings/actions/1/main.rs:38]: You mean `window.close`?
Yes, I mention that in the chapter.

Comment (bilelmoussaoui) [reply book/listings/actions/1/main.rs:38]: ah never mind me then :)

File: book/listings/actions/3/main.rs [hunk 2]
```diff
@@ -59,33 +58,32 @@ fn build_ui(app: &Application) {
...
-    );
-    action_count.connect_activate(clone!(@weak label => move |action, parameter| {
-        // Get state
-        let mut state = action
-            .state()
-            .expect("Could not get state.")
-            .get::<i32>()
-            .expect("The variant needs to be of type `i32`.");
+    let action_count = ActionEntry::builder("count")
+        .parameter_type(Some(&i32::static_variant_type()))
```
Comment (bilelmoussaoui) [inline book/listings/actions/3/main.rs:62]: Why Some? is this takes an Option??

Comment (Hofer-Julian) [reply book/listings/actions/3/main.rs:62]: Yes: https://gtk-rs.org/gtk-rs-core/stable/latest/docs/gio/struct.ActionEntryBuilder.html

If you want to make me a bit happier, then please open a PR/MR at the right place 😅

**Pull request merged**